src/components/snackbars/SnackbarNotification.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { SyntheticEvent } from 'react';
+import { Snackbar, SnackbarContent, IconButton } from '@material-ui/core';
+import ErrorIcon from '@material-ui/icons/Error';
+import { makeStyles } from '@material-ui/styles';
+import clsx from 'clsx';
+import CloseIcon from '@material-ui/icons/Close';
+
+export interface SnackbarNotificationProps {
+    variant: 'error';
```

Comment:
Set variant type as string to have the possibility to define other variants such as 'success' and 'warn'

Reply:
Done